code/src/java/pcgen/rules/context/AbstractListContext.java

```diff
@@ -60,11 +60,6 @@ void setSourceURI(URI sourceURI)
 		getCommitStrategy().setSourceURI(sourceURI);
 	}
 
-	URI getExtractURI()
```

Comment:
This is necessary for future code (some of which I have in a local branch), so please do not remove.